scripts/run_integration_tests.sh

```diff
@@ -78,12 +78,17 @@ if [ ! -f "$VMLINUX_IMAGE" ]; then
     popd
 fi
 
-VIRTIOFSD_URL="$(curl --silent https://api.github.com/repos/intel/nemu/releases/latest | grep "browser_download_url" | grep "virtiofsd-x86_64" | grep -o 'https://.*[^ "]')"
 VIRTIOFSD="$WORKLOADS_DIR/virtiofsd"
+VIRTIOFSD_DIR="virtiofsd_build"
```

Comment:
Might as well go ahead and call this QEMU_BUILD or something as @bjzhjing wants a qemu build for her testing on #69

Reply:
@rbradford @sboeuf 

Wow, cool! Thanks for providing and letting me know this while I'm struggling with the integration test with QEMU :-P  I will add it for #69.